UI/MapboxNavigationUI/Bundle.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+extension Bundle {
+    class var navigationUI: Bundle {
+        get {
+            let bundle = Bundle(for: RouteViewController.self)
+            let resourceBundlePath = "\(bundle.bundlePath)/MapboxNavigationUI.bundle"
```

Comment:
That was my initial plan but I changed my mind after reading this https://guides.cocoapods.org/syntax/podspec.html#resource_bundles
IIRC, I ran into issues with loading xib/storyboards from within the framework.
The storyboard that ends up in the bundle is a compiled version of a storyboard.
I'll take another stab at it.